ds-caselaw-ingester/lambda_function.py

```diff
@@ -13,14 +13,27 @@
 from boto3.session import Session
 from botocore.exceptions import NoCredentialsError
 from caselawclient.Client import (
+    DEFAULT_USER_AGENT,
+    MarklogicApiClient,
     MarklogicCommunicationError,
     MarklogicResourceNotFoundError,
-    api_client,
 )
+from dotenv import load_dotenv
 from notifications_python_client.notifications import NotificationsAPIClient
 
+load_dotenv()
+
+
 rollbar.init(os.getenv("ROLLBAR_TOKEN"), environment=os.getenv("ROLLBAR_ENV"))
 
+api_client = MarklogicApiClient(
```

Comment:
It's possible we could just drop this environment loading, now I've fixed the tests to not depend on the actual api_client

Reply:
whats the alternative? we'll still need to read the env vars somehow, right?

Reply:
ah right